common/scripted_effects/wc_historical_artifacts_creation_effect.txt

```diff
@@ -0,0 +1,38 @@
+create_artifact_bloodhoof_runespear_effect = {
```

Comment:
If you want it spawned, you should call this effect somewhere in this file.
<details>
<summary>Click to expand</summary>

![image](https://user-images.githubusercontent.com/46384992/189968711-9bd4492f-094e-43eb-a7c5-42aab3563f09.png)

</details>